cpp/src/io/parquet/page_data.cu

```diff
@@ -47,7 +47,7 @@ inline __device__ uint32_t rotl32(uint32_t x, uint32_t r)
   return __funnelshift_l(x, x, r);  // (x << r) | (x >> (32 - r));
 }
 
-inline __device__ int rolling_index(int index) { return index & (non_zero_buffer_size - 1); }
+constexpr int rolling_index(int index) { return index & (non_zero_buffer_size - 1); }
```

Comment:
Please also move this along with the function/variables up to `block_size` above into an anonymous namespace.

Reply:
Will do.  It seems the `rotl32()` function isn't actually used, so I'm removing it as well.